docs/UserGuide.md

```diff
@@ -80,11 +80,20 @@ Format: `help`
 Adds a customer to ReadyBakey’s contact list
 
 Format: `addp n/NAME p/PHONE e/EMAIL a/ADDRESS [r/REMARK] [t/TAG]…`
-* Names must be between 2 to 50 characters long and must only contain alphanumeric characters and spaces.
-* The length of the `PHONE` must be 3 to 15 numbers
-* The length of the `EMAIL` must be between 6 to 50 characters
-* The length of the `ADDRESS` must be between 6 to 100 characters
-* The length of the `REMARK` must be less than or equal to 70 characters
+* Names must be between 2 and 50 characters long and must only contain alphanumeric characters and spaces.
+* The length of the `PHONE` must be between 3 and 15 numbers.
```

Comment:
I saw that the phone number cant have spaces iirc, (in testcases), should we add that in here?

Reply:
Ok! Updated this.